feat: add image generation providers (#75)
prabirshrestha/ai.rs: crates/ai/README.md

- `openai-responses`
- `anthropic-messages`

> The active built-in image generation API is:

Author: `API is` -> `APIs are`
```suggestion
The active built-in image generation APIs are:
```

Author: Fixed in f9f9ffda.
